companies/volkswagen-ag.json

```
@@ -0,0 +1,21 @@
+{
+    "slug": "volkswagen-ag",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "Volkswagen AG",
+    "address": "Volkswagen AG\nPrivacy Team\nBerliner Ring 2\n38440 Wolfsburg\nGermany",
```

The company name and an ATTN line are automatically added by our generator (see https://github.com/datenanfragen/data#addresses).

```suggestion
    "address": "Berliner Ring 2\n38440 Wolfsburg\nGermany",
```

---

I see, sorry I missed that! This is taken care of now.